experiments/AtmosLES/squall_line.jl

```diff
@@ -122,8 +122,19 @@ end
   Read the original squall sounding
 """
 function read_sounding()
+
+    # Artifact creation is not thread-safe
+    #      https://github.com/JuliaLang/Pkg.jl/issues/1219
+    # To avoid race conditions from multiple jobs running this
+    # driver at the same time, we must store artifacts in a
+    # separate, job-unique, folder.
+
+    job_id = ENV["BUILDKITE_AGENT_META_DATA_JOBID"]
```

Comment:
🤔 What if this isn't running under Buildkite?

Maybe just use [`mktempdir`](https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.mktempdir-Tuple{AbstractString})?

Reply:
That's exactly what I was thinking. I forgot about `mktempdir`, good idea!